scripts/container-only/wkdev-setup-vscode

```diff
@@ -96,15 +133,78 @@ install_extensions() {
     install_extension ms-python.python "Python support" true
 }
 
+# These are VERY helpful for WebKit development, but we won't override existing settings if the user already has them.
+default_settings() {
+    if argsparse_is_option_set "proprietary"; then
+        VSCODE_CONFIG_PATH=${HOME}/.config/Code/User/
+    else
+        VSCODE_CONFIG_PATH=${HOME}/.config/VSCodium/User/
+    fi
+    if [[ ! -e "${VSCODE_CONFIG_PATH}/settings.json" ]]; then
+        mkdir -p "${VSCODE_CONFIG_PATH}"
+        tee "${VSCODE_CONFIG_PATH}/settings.json" << HERE
+{
+    "clangd.arguments": [
+        "-header-insertion=never"
+    ],
+    "terminal.integrated.scrollback": 1000000000,
+    "editor.renderWhitespace": "trailing",
+    "window.menuBarVisibility": "toggle",
+    "workbench.colorCustomizations": {
+    "editorWhitespace.foreground": "#FF0000",
+    "editorWhitespace.background": "#FF0000"
+},
+"terminal.integrated.allowChords": false,
+}
+HERE
+        echo "Installed default VSCode settings to ${VSCODE_CONFIG_PATH}."
+    else
+        echo "There was already a VSCode settings.json (${VSCODE_CONFIG_PATH}), skipping."
+    fi
+}
+
+install_xdg() {
+    tee /host/${HOME}/.local/share/applications/codium.desktop << HERE
+[Desktop Entry]
+Name=VSCode WKDev
+Comment=Code Editing. Redefined.
+GenericName=Text Editor
+Exec=bash -c "source /home/${USER}/.bashrc; wkdev enter -I -e -- $CODE_EXEC"
```

Comment:
```suggestion
Exec=bash -c "source /home/${USER}/.bashrc; wkdev-enter --exec --no-interactive -- $CODE_EXEC --unity-launch %F"
```

code takes a list of files.

explict args are more readable.

Reply:
This makes a strong assumption users configure bash, which just isn't true.

This needs to be changed to be an absolute path to `wkdev-enter` on the host. I'm not sure how to get that information off-hand.